clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -182,7 +182,10 @@
     INVALID_TXN_STATE(48,
             new InvalidTxnStateException("The producer attempted a transactional operation in an invalid state")),
     INVALID_PID_MAPPING(49,
-            new InvalidPidMappingException("The PID mapping is invalid"));
+            new InvalidPidMappingException("The PID mapping is invalid")),
+    INVALID_TRANSACTION_TIMEOUT(50,
+            new InvalidSessionTimeoutException("The transaction timeout is larger than the maximum value allowed by the broker " +
```

Comment:
Adding the exception and also edited https://confluentinc.atlassian.net/browse/CPKAFKA-616 to track doc changes since it is "semi-"treated as public apis.